static/arguments.yml

```diff
@@ -29,6 +29,28 @@
     type: Hash
     default: N/A
     description: "[System arguments](/system-arguments)"
+- component: Layout
+  source: https://github.com/primer/view_components/tree/main/app/components/primer/alpha/layout.rb
+  parameters:
+  - name: stacking_breakpoint
+    type: Symbol
+    default: "`:md`"
+    description: When the `Layout` should change from rows into columns. One of `:lg`,
+      `:md`, or `:sm`.
+  - name: first_in_source
+    type: Symbol
+    default: "`:sidebar`"
+    description: Which element to render first in the HTML. This will change the keyboard
```

Comment:
Shouldn't this have a "one of" part?

Reply:
Yep! https://github.com/primer/view_components/pull/889/commits/8782f1d5f8895d56f25542e072efc5338cba42b6

Reply:
Yep! https://github.com/primer/view_components/pull/889/commits/8782f1d5f8895d56f25542e072efc5338cba42b6